src/components/common/TokenAmount/index.tsx

```diff
@@ -12,16 +12,18 @@ const TokenAmount = ({
   tokenSymbol,
   direction,
   fallbackSrc,
+  maxLength,
 }: {
   value: string
   decimals?: number
   logoUri?: string
   tokenSymbol?: string
   direction?: TransferDirection
   fallbackSrc?: string
+  maxLength?: number
 }): ReactElement => {
   const sign = direction === TransferDirection.OUTGOING ? '-' : ''
-  const amount = decimals !== undefined ? formatVisualAmount(value, decimals) : value
+  const amount = decimals !== undefined ? formatVisualAmount(value, decimals, undefined, maxLength) : value
```

Comment:
`formatVisualAmount` always drops amounts `< 0.00001` and just shows "< .00001".
We could consider giving the TokenAmount component a `precise: boolean` prop and use `formatAmountPrecise` instead.
That's what we use within the `SendAmountBlock` component.
This would make it consistent to what we show in the Review step of transaction creation.

Reply:
Good point, `formatAmountPrecise` works for both big and small numbers and is consistent with  `SendAmountBlock` 👍